app/core/config.py

```diff
@@ -18,3 +19,19 @@
 SECRET_KEY = os.getenv("SECRET_KEY")
 
 ACCESS_TOKEN_EXPIRE_MINUTES = 60 * 24 * 8  # 60 minutes * 24 hours * 8 days = 8 days
+
+# Configs for sending emails
+MAIL_SERVER = os.getenv("MAIL_SERVER")
+MAIL_PORT = int(os.getenv("MAIL_PORT")) or 465
+MAIL_USERNAME = os.getenv("MAIL_USERNAME")
+MAIL_PASSWORD = os.getenv("MAIL_PASSWORD")
+MAIL_SENDER = os.getenv("MAIL_SENDER")
+if tls := os.getenv("MAIL_USE_TLS"):
+    if tls == "True":
+        MAIL_USE_TLS = True
+    elif tls == "False":
+        MAIL_USE_TLS = False
+        warning("Setting MAIL_USE_TLS to `False`")
```

Comment:
Python dotenv has a feature to cast variables into booleans, but that might be done in another PR